Project.toml

```diff
@@ -3,6 +3,7 @@ uuid = "42fd0dbc-a981-5370-80f2-aaf504508153"
 version = "0.9.0"
 
 [deps]
+IncompleteLU = "40713840-3770-5561-ab4c-a76e7d0d7895"
```

Comment:
We have a file for test dependencies, https://github.com/JuliaLinearAlgebra/IterativeSolvers.jl/blob/master/test/Project.toml, we only need that dependency for the test, right?

Reply:
I just checked the "gmres.jl" and found it use LU instead of the IncompleteLU I added in the dependency，shall I removed it?

Reply:
It makes sense to have a non-exact preconditioner in the test, or? Perhaps just move the 
```
IncompleteLU = "40713840-3770-5561-ab4c-a76e7d0d7895"
```
line to https://github.com/JuliaLinearAlgebra/IterativeSolvers.jl/blob/master/test/Project.toml

Reply:
ok